test/e2e/all_in_one_test.go

```diff
@@ -69,7 +69,10 @@ const (
 	adminAPIWait = time.Minute * 2
 )
 
-var imageOverride = os.Getenv("TEST_KONG_CONTROLLER_IMAGE_OVERRIDE")
+var (
+	imageOverride = os.Getenv("TEST_KONG_CONTROLLER_IMAGE_OVERRIDE")
+	imageLoad     = os.Getenv("TEST_KONG_CONTROLLER_IMAGE_LOAD")
```

Comment:
I know the previous var was missing any godoc, but now that we have two similar vars it seems like a good time to add some godoc here to point out upfront what they're for.